### PR TITLE
chore: use node 14

### PR DESCRIPTION
**Pull request by chmelevskij**
This is matching PR to use node 14 similar to how it's done in configurator. This also adds `.nvmrc` and `engines` to match the configurator.

### CODE REVIEW

File: images/bf_logo_white.svg
Comment (mikeller) [inline images/bf_logo_white.svg:1]: What is this change for?

**Review by mikeller — COMMENTED** [review]
Have you tested if video playback is working with the new node version? This is often the sticky point for version compatibility with blackbox explorer, as external codecs are used.

File: images/bf_logo_white.svg
Comment (chmelevskij) [reply images/bf_logo_white.svg:1]: No idea, will remove.

**Comment by McGiverGim** [comment]
I've tested it under Windows and it seems video playback works. I'm not totally sure because it is not too smooth but I think this happens too with older version.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@chmelevskij please rebase. I think this should be merged soon to be in sync with configurator.

**Comment by ctzsnooze** [comment]
@chmelevskij it would be great to action this - any chance you could review the conflicts and re-check it is OK?

**Comment by chmelevskij** [comment]
Hey, will try over the weekend. Want ot match the setup with configurator first.

**Comment by chmelevskij** [comment]
Updated with changes from configurator still need to verify the builds though.

**Comment by haslinghuis** [comment]
@chmelevskij it's not building. Any update on this?

File: package.json
Comment (haslinghuis) [inline package.json:42]:
```suggestion
    "gulp-yarn": "^3.0.0",
```

Comment (haslinghuis) [reply package.json:42]: No OSX here. But works on Linux.

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@chmelevskij seems some files does not belong here?

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
